integration-test/utils/datafactory/schemas/codeofconduct.schema.ts

```diff
@@ -0,0 +1,41 @@
+import { pageSchema } from './page.schema';
+import { linkSchema } from './link.schema';
+
+export const codeofconductSchema = {
+  $ref: '#/definitions/pageSchema',
+  definitions: {
+    codeofconductSchema: {
```

Comment:
This object will change soon after the changes done by @nverbic maybe you should wait to avoid rework.